src/pages/accessibility-commitment.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import Layout from "../components/layout"
+import HTMLContent from "../components/HTMLContent"
+import { useAccessibilityCommitment } from "../fetchHooks/useAccessibilityCommitment"
+import Seo from "../components/seo"
+
+
+const AccessibilityCommitment = () => {
+  const { frontmatter, html } = useAccessibilityCommitment()
+  const { heroHeading } = frontmatter
+  return (
+    <Layout>
+      <Seo title="ADA Statement | Protect your Properties with Landlord Tech" description="Ensure protection of your rental properties with the best pet management platform from Landlord Tech." />
```

Comment:
should this texts added from admin instead of hardcoding here?

Reply:
Yes, but when I tried it gave me an error (because I don't have those vars in the CMS). But I will do it. The same happened to me with the links in the footer (I added a new one, and I tried to do as the others, Privacy Policy and Terms and Conditions, but it gave me an error because that info is not in the CMS) I don't know if there another way to do it. If yes, feel free to share it with me so I can implement it in that way.

Reply:
that's probably you have changed something in the setting during this period which affected if you say that you're doing the same as was done before but it's not applying. You should investigate and fix the bug. I don't know what can it be I should investigate myself, but this is important to fix, because there are going to be future request to add a new pages or sections with all admin editable.

Reply:
I will take a look, thanks

Reply:
Fixed